src/main/java/intellistart/interviewplanning/controllers/dto/DashboardMapDto.kt

```diff
@@ -0,0 +1,73 @@
+package intellistart.interviewplanning.controllers.dto
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import intellistart.interviewplanning.model.candidateslot.CandidateSlot
+import intellistart.interviewplanning.model.dayofweek.DayOfWeek
+import intellistart.interviewplanning.model.interviewerslot.InterviewerSlot
+import intellistart.interviewplanning.model.week.WeekService
+import java.time.LocalDate
+import java.util.*
+
+/**
+ * Dto object for representation all candidate, interviewer slots
+ * and bookings for a certain week.
+ */
+data class DashboardMapDto(
+    @JsonIgnore
+    val weekService: WeekService,
+    val weekNum: Long,
+    val dashboard: HashMap<LocalDate, DashboardDto>
```

Comment:
can you use the interface here?

Reply:
do you mean the map interface instead of hashmap? or the DashboardMapDto class?

Reply:
map interface